kubejs/server_scripts/enigmatica/kubejs/normal/recipetypes/mekanism/metallurgic_infusing.js

```diff
@@ -6,10 +6,10 @@ events.listen('recipes', (event) => {
     var data = {
         recipes: [
             {
-                output: Item.of('compactmachines:wall', 64),
+                output: Item.of('compactmachines:wall', 32),
                 inputItem: '#forge:storage_blocks/ender',
```

Comment:
So close, storage_blocks -> ingots

Reply:
look again, those are ender blocks (4 pearls) not enderium. Mek uses refined obsidian so I felt like enderium and refined obsidian was a bit much. I think of it as an alternative gate per say

Reply:
Ah of course, my bad :D